README.md

```diff
@@ -1 +1,132 @@
+
+We encourage instructors to customize this representations based on their instructional plan and the actual skills being covered in their course or bootcamp.
+
+Repositories for other frameworks:
+
+-   MIT’s Digital Humanities Framework: https://github.com/ccnmtl/diglit-dh-badge
+
+-   ACRL’s Framework: https://github.com/ccnmtl/diglit-acrl-badge
+
+**Installation**
+------------------------------------------------------------------
+
+1.  Choose the the framework that is most appropriate for your instructional context. Please see Choosing Your Framework in our[Digital Literacy Instructional Design Process](https://docs.google.com/document/d/1dChIUmHyUDAgzAV5Wt3h8QZlkrqedgSnR0bsoQz2U9w/edit?usp=sharing).
+
+    a.  [The ACRL Framework](https://docs.google.com/spreadsheets/d/1Im8N3deuB-bLaRadCevr2vpb-CjO_EThNtvV-mdBpwY/edit?usp=sharing)
+
+    b.  [Eshet’s Cognitive Skills Framework](https://docs.google.com/spreadsheets/d/12kCXyRT_QJ7gI0yYF4TMSK-Dlie3Yj1fc-vH0_5giCI/edit?usp=sharing)
+
+    c.  [MIT’s Digital Humanities Framework](https://docs.google.com/spreadsheets/d/1ZjeoO_hf3EbodS1-SUxg9jrViDcENWSGBrefv702TJI/edit?usp=sharing)
+
+2.  Create a repository for your Digital Literacy Competency Calculator in your GitHub account (*please see below for instructions on how to do this*).
+
+3.  Make a copy of the Google Sheets DLCC template for the framework you selected:
+
+![Screenshot of menu for copying a Google Sheet](media/images/Make_A_Copy_Google_Sheet.png)
+
+4.  Customize the spreadsheet to match your instructional design. Please see our [Digital Literacy Instructional Design Process](https://docs.google.com/document/d/1dChIUmHyUDAgzAV5Wt3h8QZlkrqedgSnR0bsoQz2U9w/edit?usp=sharing).
+
+5.  When you are happy with the spreadsheet, download it as a Comma-Separated Values and name the file “**standards**”.
+
+![Screenshot of menu to download Google Sheet as .csv](media/images/Download_As_CSV_Google_Sheet.png)
+
+6.  Go to the “csv” folder inside the “images” folder in your repository. Drag the file “standards.csv” into the “csv” folder. It should replace the existing “standards.csv” file.
+
+7.  Describe your change and create a new branch for this commit and start a pull request. ([**What is a pull request?**](https://help.github.com/articles/about-pull-requests/))
+
+![Screenshot of proposing a change in GitHub](media/images/Propose_Changes_GitHub.png)
+
+8.  Go to the “index.html” file in your repository and edit the Title (line 203) and descriptions (line 206) to suit your context. Describe your change and create a new branch for this commit and start a pull request.
+
+    a.  Note: All other descriptions (for lenses, pop up box copy, etc.) are determined by your “standards.csv” file. To make further changes to description, you must make changes in your Google Sheet and upload a new “standards.csv” file with your changes. We discourage editing in the standards.csv file directly.
+
```

Comment:
I think you need a 'the' here: "To make further changes to *the* description, ..."